src/core/action-creator.ts

```diff
@@ -102,6 +102,10 @@ export function createActions(flux: FluxCapacitor) {
                 .findIndex((index) => SearchAdapter.refinementsMatch(payload, <any>navigation.refinements[index], navigation.range ? 'Range' : 'Value')) === -1;
             },
             msg: 'refinement is already selected'
+          }, {
+            // tslint:disable-next-line max-line-length
+            func: ({ range }, state) => range ? !(Selectors.rangeHighest(state, field) === high && Selectors.rangeLowest(state, field) === valueOrLow) : true,
```

Comment:
can make this `!!range && !(/* other logic */)`

Reply:
I'm not sure that's equivalent. The original is `true` when `range` is falsey. I think you want `!range || !(/* other logic*/)`. Also, you might benefit from DeMorgan's law for the other logic.